### PR TITLE
Bump upx

### LINKED ISSUE #221: Installing code on Big Sur doesn't seem to do anything
❯ curl -sL https://github.com/karimra/gnmic/raw/master/install.sh | sudo bash
Downloading https://github.com/karimra/gnmic/releases/download/v0.4.3/gnmic_0.4.3_darwin_x86_64.tar.gz
Preparing to install gnmic 0.4.3 into /usr/local/bin
gnmic installed into /usr/local/bin/gnmic
bash: line 173:  6287 Killed: 9               $GNMIC_INSTALL_DIR/$BINARY_NAME version

When I run the command I always get this..

[1]    6356 killed     gnmic
╭─ ~/Downloads/junos-qfx ▓▒░································································································░▒▓ KILL ✘  at 23:44:31 
╰─

### PR DESCRIPTION
**Pull request by hellt**
fixes #221 

upx 3.96 promises to fix the issue with Big Sur

To install upx 3.96 I had upgrade the runners system to run ubuntu 20.04. Anyhow, soon it will become ubuntu-latest, so should be all fine

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -27,7 +27,7 @@ jobs:
           go-version: 1.13
 
       - name: Install upx
-        run: sudo apt update && sudo apt install upx -y
+        run: sudo apt update && sudo apt install libucl1 && curl -L http://archive.ubuntu.com/ubuntu/pool/universe/u/upx-ucl/upx-ucl_3.96-2_amd64.deb -o upx.deb && dpkg -i upx.deb
```
Comment (karimra) [inline .github/workflows/release.yml:30]: sudo for the last cmd? `sudo dpkg -i upx.deb`

Comment (hellt) [reply .github/workflows/release.yml:30]: yes, will push a fix

Comment (hellt) [reply .github/workflows/release.yml]: fixed in 209cf27

**Comment by karimra** [comment]
lgtm

**Pull request merged**